__init__.py

```diff
@@ -12,19 +12,18 @@
 
 import git
 
-import zeekpkg.template # pylint: disable=import-error
```

Comment:
`import-error` and `no-self-use` below go away here, but I don't see it in the `.pylintrc`. Were they not necessary to begin with? I can confirm that pylint runs fine on the result over here, so it seems that way.

Reply:
Yes, the code seemed fine so I removed the override.